spanner/google/cloud/spanner/_helpers.py

```diff
@@ -20,9 +20,8 @@
 import six
 
 from google.gax import CallOptions
-from google.protobuf.struct_pb2 import ListValue
-from google.protobuf.struct_pb2 import Value
 from google.cloud.proto.spanner.v1 import type_pb2
+from google.cloud.spanner_v1 import types
```

Comment:
Is this a documented change to the Spanner API?  System tests have been passing all along with the non-Spanner-specific types protos, which *were* the documented forms previously.

Reply:
Both paths work. We updated the autogen to alias to a more user-friendly path for cases where we actually expose this directly to users.